# A 6D vector encapsulating linear and angular information in an R^3 space

geometry_msgs/Vector3 linear
geometry_msgs/Vector3 angular
